src/battle_script_commands.c

```diff
@@ -14309,6 +14309,45 @@ static void Cmd_recoverbasedonsunlight(void)
             else // not sunny weather
                 gBattleStruct->moveDamage[gBattlerAttacker] = GetNonDynamaxMaxHP(gBattlerAttacker) / 4;
         }
+        else // B_TIME_OF_DAY_HEALING_MOVES == GEN_2
+        {
+            u32 healingModifier;
+            u32 time = GetTimeOfDay();
+            
+            switch (GetMoveEffect(gCurrentMove))
+            {
+            case EFFECT_MOONLIGHT:
+                if (time == TIME_NIGHT || time == TIME_EVENING)
+                {
+                    healingModifier = 2;
+                }
```

Comment:
```suggestion
                    healingModifier = 2;
```

you don't need brackets if it's just one line. applies to the rest